lib/Routing/Staticroute/Router.php

```diff
@@ -245,7 +245,7 @@ protected function doMatch(string $pathinfo, Request $request = null): array
                 // to determine if a call to an action was made through a staticroute or not
                 // more on that infos see Pimcore_Controller_Action_Frontend::getRenderScript()
                 $routeParams['pimcore_request_source'] = 'staticroute';
-                $routeParams[ElementListener::FORCE_ALLOW_PROCESSING_UNPUBLISHED_ELEMENTS] = $this->config['routing']['allow_processing_unpublished_fallback_document'];
+                $routeParams[ElementListener::FORCE_ALLOW_PROCESSING_UNPUBLISHED_ELEMENTS] = true;
```

Comment:
Are these routeParams used anywhere? And if the value is now true anyways, couldn't we just remove this all together? Same as [here](https://github.com/pimcore/pimcore/blob/11.x/lib/Mail.php#L678) in den Mail.php right then?
[EditableUsageResolver](https://github.com/pimcore/pimcore/blob/11.x/lib/Document/Editable/EditableUsageResolver.php#L52)